### PR TITLE
doc: Update minimum required Qt

### PR DESCRIPTION
**Pull request by fanquake**
After #14725 the minimum required version of Qt is 5.2.

### CODE REVIEW

File: doc/dependencies.md [hunk 1]
```diff
@@ -3,15 +3,15 @@ Dependencies
...
 | --- | --- | --- | --- | --- | --- |
-| Berkeley DB | [4.8.30](http://www.oracle.com/technetwork/database/database-technologies/berkeleydb/downloads/index.html) | 4.8.x | No |  |  |
+| Berkeley DB | [4.8.30](https://www.oracle.com/technetwork/database/database-technologies/berkeleydb/downloads/index.html) | 4.8.x | No |  |  |
 | Boost | [1.64.0](https://www.boost.org/users/download/) | [1.47.0](https://github.com/bitcoin/bitcoin/pull/8920) | No |  |  |
 | Clang |  | [3.3+](https://llvm.org/releases/download.html) (C++11 support) |  |  |  |
 | D-Bus | [1.10.18](https://cgit.freedesktop.org/dbus/dbus/tree/NEWS?h=dbus-1.10) |  | No | Yes |  |
 | Expat | [2.2.5](https://libexpat.github.io/) |  | No | Yes |  |
 | fontconfig | [2.12.1](https://www.freedesktop.org/software/fontconfig/release/) |  | No | Yes |  |
-| FreeType | [2.7.1](http://download.savannah.gnu.org/releases/freetype) |  | No |  |  |
+| FreeType | [2.7.1](https://download.savannah.gnu.org/releases/freetype) |  | No |  |  |
```
Comment (hebasto) [inline doc/dependencies.md]:
```
| FreeType | [2.7.1](https://download.savannah.gnu.org/releases/freetype) |  | No |  | [No](https://github.com/bitcoin/bitcoin/blob/master/depends/packages/qt.mk#L33) |
```

And a similar change for `zlib`?

**Review by hebasto — CHANGES_REQUESTED** [review]
Concept ACK.

**Comment by MarcoFalke** [comment]
ACK b7bee6af76e076e619b4ce5f165e81c6b4c52b2d

**Comment by laanwj** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/14909/commits/b7bee6af76e076e619b4ce5f165e81c6b4c52b2d

**Pull request merged**